ReadMe.md

```diff
@@ -1,4 +1,4 @@
 This repo contains the source material and build tools for: http://www.yaml.org
 
 The upkeep of http://www.yaml.org is a community project. You can easily
-contribute! See [[Contributing.md]] for details.
+contribute! See [the Contribution guide](./Contributing.md) for details.
```

Comment:
Don't do `./`, just do `Contributing.md` :) It helps sites other than Github render content a bit better (sounds strange, I know).

```suggestion
contribute! See [the Contribution guide](Contributing.md) for details.
```